File: repl/adrianus/python3/instructions.py
class Instructions:
    def __init__(self):
        # a list of bytes
        # List<Byte>
        self.instructions = []

    def size(self):
        return len(self.instructions)

    def get(self, i):
        return self.instructions[i]

    def add(self, ins):
        self.instructions.append(ins)

    def __str__(self):
        out = ""
        for instruction in self.instructions:
            out += str(instruction) + "\n"
        return out

File: repl/adrianus/java/ad/src/main/java/com/ad/vm/Instructions.java
package com.ad.vm;

import java.util.Arrays;

public class Instructions {
    private byte[] bytes;
    private int n = 0;

    public int size() {
        return bytes.length;
    }

    public byte get(int i) {
        return bytes[i];
    }

    public void add(byte instruction) {
        bytes[n++] = instruction;
    }

    @Override
    public String toString() {
        return "Instructions{" +
                "bytes=" + Arrays.toString(bytes) +
                ", n=" + n +
                '}';
    }
}
